package.json

```diff
@@ -26,7 +26,7 @@
     "eslint-plugin-import": "^2.25.0",
     "eslint-plugin-prettier": "^3.3.0",
     "eslint-plugin-react": "^7.26.0",
-    "babel-eslint": "^10.0.0",
+    "@babel/eslint-parser": "^7.5.4",
```

Comment:
it seems that the package needs some other dependencies and configuration https://www.npmjs.com/package/@babel/eslint-parser

Reply:
Seeing this makes me think that the changes i made here https://github.com/inveniosoftware/react-invenio-deposit/pull/551/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R68  should've been made in this repository instead.

Reply:
made another PR which should include all the necessary changes https://github.com/inveniosoftware/eslint-config-invenio/pull/4/files

Reply:
do we close this one?